engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -3715,6 +3718,50 @@ DTConfiguration getLaunchAppPackageProperties(AppPackage ap, ConfigPackage cp, L
     return launchProperties;
   }
 
+  private List<AppInfo> getAppsFromPackageAndConfig(AppPackage ap, ConfigPackage cp, boolean onlyConfigApps)
+  {
+    if (cp == null) {
+      return ap.getApplications();
+    }
+
+    File src = new File(cp.tempDirectory(), "app");
+    File dest = new File(ap.tempDirectory(), "app");
+
+    if (!src.exists()) {
+      return ap.getApplications();
+    }
+
+    if (onlyConfigApps) {
+      try {
+        FileUtils.cleanDirectory(dest);
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+    } else {
+      for (File app: src.listFiles()) {
```

Comment:
Please check the style of your IDE. There should be a space before this colon. Also check all other occurrences.